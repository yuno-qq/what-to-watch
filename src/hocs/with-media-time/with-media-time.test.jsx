import React from "react"
import {describe, test, expect} from "@jest/globals"
import renderer from "react-test-renderer"

import withMediaTime from "./with-media-time.jsx"


describe(`withMediaTime() snapshots:`, () => {
  test(`should render correctly`, () => {
    const MockComponentWrapped = withMediaTime(() => <div className="MOCK_COMPONENT"></div>)
    const tree = renderer.create(<MockComponentWrapped />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
