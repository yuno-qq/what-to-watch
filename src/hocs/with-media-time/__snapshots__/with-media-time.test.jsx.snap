// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withMediaTime() snapshots: should render correctly 1`] = `
<div
  className="MOCK_COMPONENT"
/>
`;
